docs/high-availability/Vagrantfile

```diff
@@ -0,0 +1,51 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  fedora = 'hansode/fedora-21-server-x86_64'
+  script = "provision.sh"
+  #discovery_url = `wget https://discovery.etcd.io/new?size=3 -O -`
+  discovery_url = "dummy"
```

Comment:
the discovery bit is not used in this recipe.  The reason being that we spin up a single node etcd in a libeled to avoid any cyclic dependencies, as this is really supposed to be a minimal impl of the podmAster